src/main/webapp/WEB-INF/i18n/errors_en.properties

```diff
@@ -30,5 +30,6 @@ typeMismatch.thumbnail=Valid thumbnail format: PNG
 typeMismatch.java.lang.Integer={0} must be a number
 formatHint.Integer=Number (e.g. "5000")
 image.too.small=The image width must be at least 640px
+image.too.big=The image size is too big
```

Comment:
```suggestion
file.size.too.big=The file size is too big
```

Reply:
> @tomaszsmy Do you think we should also reduce from 10MB to 4MB in `Image.java`?
> 
> https://github.com/elimu-ai/webapp/blob/bf77a8f30d9996ba92ea917db68e5bb6ffceb8c0/src/main/java/ai/elimu/model/content/multimedia/Image.java#L19

I think that the change may cause error if in table are any data which has size > 4MB